packages/terra-graphs-docs/src/terra-dev-site/doc/example/Graphs/BarGraph/BasicBarGraphs/AxisInfoTextLabels.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import BarGraph from '@cerner/terra-graphs/lib/components/Bar/BarGraph';
+import utils from '@cerner/carbon-graphs/lib/js/helpers/utils';
+import '@cerner/terra-graphs/lib//components/Dev.module.scss';
+import barDefault from '@cerner/terra-graphs-docs/lib/example-datasets/graphConfigObjects/Bar/barDefault';
+import data from '@cerner/terra-graphs-docs/lib/example-datasets/dataObjects/Bar/axisInfoTextLabelsData';
+
+/*
+Please refer documentation below to see graphConfig and data objects.
```

Comment:
Nit: `Please refer to the documentation below to see the graphConfig and data objects`
(Same for the other files)

Reply:
> Nit: `Please refer to the documentation below to see the graphConfig and data objects`
> (Same for the other files)

Reply:
Changed [message](https://github.com/cerner/terra-graphs/pull/106/commits/1a7e6e62e5536262a60f6472fccbb52536e54fe6#diff-db9d8f32df60056dc62526adda60a897e356e4979a5c7378a281e75798823dd3R9) for all the examples.